Eat_and_Greet/App.tsx

```diff
@@ -1,20 +1,37 @@
-import { NavigationContainer } from '@react-navigation/native';
-import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import { NavigationContainer, DefaultTheme } from "@react-navigation/native";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
 
-import HomeScreen from './src/screens/HomeScreen';
-import LoginScreen from './src/screens/LoginScreen';
-import OnboardingScreen from './src/screens/OnboardingScreen';
+import HomeScreen from "./src/screens/HomeScreen";
+import LoginScreen from "./src/screens/LoginScreen";
+import OnboardingScreen from "./src/screens/OnboardingScreen";
+import ForumsScreen from "./src/screens/ForumsScreen";
+import ProfileScreen from "./src/screens/ProfileScreen";
+import SearchScreen from "./src/screens/SearchScreen";
+import SettingsScreen from "./src/screens/SettingsScreen";
 
+import HeaderBackButton from "./src/comps/HeaderBackButton";
+
+import { navHeaderStyles } from "./src/style";
 
 const Stack = createNativeStackNavigator();
 
 export default function App() {
   return (
     <NavigationContainer>
-      <Stack.Navigator>
+      <Stack.Navigator
+        screenOptions={{
+          ...navHeaderStyles,
+          headerBackTitleVisible: true,
+          headerLeft: () => <HeaderBackButton />,
+        }}
+      >
         <Stack.Screen name="Login" component={LoginScreen} />
```

Comment:
I like this, worth considering doing this in a for loop with an array of tuples or something. If not done now, can make a ticket to do later.

Reply:
used a map function & tuple array to change